pkg/common/logger.go

```diff
@@ -10,6 +13,15 @@ type loggerContextKey string
 
 const loggerContextKeyVal = loggerContextKey("logrus.FieldLogger")
 
+var secrets []string
```

Comment:
The `secrets` are kept across all runs during the 'lifetime' of act.
I think it would be better to scope this to a running `job`. You can put the secrets into the `RunContext` and add the `AddSecret` function over there.

Otherwise having one job which sets `hello` as a secret and a second job which outputs `hello` as a value will have that masked. I think that would be wrong.

Reply:
But it will make it more complex to connect the log formatter to the `RunContext`.